app/main.py

```diff
@@ -2,0 +1,19 @@
+from app.cafe import Cafe
+from app.errors import (VaccineError,
+                        NotWearingMaskError)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor=visitor)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    if masks_to_buy:
```

Comment:
Good job on not comparing a variable to `0` when there is no need. This is a good practice for code efficiency.